tests/test_visitors/test_tokenize/test_comments/conftest.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+
+from os import chmod
+
+import pytest
+
+TEMP_FOLDER = 'tmp'
+MODE_EXECUTABLE = 0o755
+MODE_NON_EXECUTABLE = 0o644
+
+
+@pytest.fixture()
+def make_file(tmp_path):
+    """Fixture to make a temporary executable or non executable file."""
+    def factory(filename, file_content, is_executable):
```

Comment:
Let's type these two functions!

Reply:
_Two_ functions ? Typing `factory` makes sense but do you want to type `make_file` as well ?

Reply:
Yes, you are right. Only `factory`. Sorry!